vimeo/VimeoNetworking
# TVOS-151: Sample project

# Ticket(s)

https://vimean.atlassian.net/browse/TVOS-151

# Ticket Summary

Clean up and improve the sample project.  Provide a taste of how to do things with `VimeoNetworking`.

# Implementation Summary

- On launch, app tries to load an account, and falls back to client credentials if it didn’t/couldn’t load one.
- If client credentials gets a 40X response, we prompt the user to double check their client identifier and secret, which is the most likely culprit.
- In MasterViewController, we listen for account changed notifications and update the table view of video names to suit.  Logged in shows /me/videos and logged out shows /channels/staffpicks/videos.
- Log In/Log Out button in nav bar launches code grant auth URL if logged out or logs out if logged in.

# How to test

Get a clientID/secret from the api site and plug them in.  Make sure to set up the redirect URL things too (see readme).
